### PR TITLE
Companies acceptance tests clean up

### PR DESCRIPTION
**Pull request by yeahfro**
Before starting the CDMS reference display on company details page I spotted some opportunities to clean up.

Included:
- removing redundant code
- consistency with other features
- reinstate companies search test

### CODE REVIEW

File: test/acceptance/features/companies/page-objects/Company.js [hunk 1]
```diff
@@ -106,7 +108,8 @@ module.exports = {
               .waitForElementPresent('@continueButton')
               .click('@continueButton')
               .waitForElementPresent('@pageHeading')
-              .api.perform((done) => {
+              .api
+              .perform((done) => {
```
Comment (teneightfive) [inline test/acceptance/features/companies/page-objects/Company.js]: Does this need a new line? Seemed to make more sense before as each new line is a method call. `.api` looks a little lost on its own...

File: test/acceptance/features/companies/page-objects/Company.js [hunk 2]
```diff
@@ -180,29 +214,61 @@ module.exports = {
                 })
               })
               .perform((done) => {
+                this.getRadioOption('headquarter_type', (result) => {
```
Comment (teneightfive) [inline test/acceptance/features/companies/page-objects/Company.js]: These changes feel like there's starting to get a bit repetitive. I don't really know nightwatch that well yet but wonder if there's something that can be done to improve this.

Probably more likely a follow-up change though so not blocking.

File: test/acceptance/features/companies/save.feature [hunk 3]
```diff
@@ -1,28 +1,34 @@
-@companies-create
+@companies-save
```
Comment (teneightfive) [inline test/acceptance/features/companies/save.feature]: What's the purpose of this rename? Feels like its still part of a create journey?

File: test/acceptance/features/companies/page-objects/Company.js (hunk 2 above)
Comment (feedmypixel) [reply test/acceptance/features/companies/page-objects/Company.js]: @teneightfive FYI there are a number of state helpers coming that will improve all this repetition

File: test/acceptance/commands/getRadioOption.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/**
+ * Get a random option from a radio button list
+ * @param selector
+ * @param callback
+ */
+exports.command = function getRadioOption (name, callback) {
+  return this.elements('css selector', `input[name="${name}"] + label span`, (result) => {
+    const random = Math.floor(Math.random() * (result.value.length - 1))
```
Comment (feedmypixel) [inline test/acceptance/commands/getRadioOption.js]: FYI I have wrapped a check around this for if (result.value.length) { and then log an error on the else for getListOption

**Pull request merged**